sh_scrapy/extension.py

```diff
@@ -42,10 +42,11 @@ def __init__(self, crawler):
         self.crawler = crawler
         self.logger = logging.getLogger(__name__)
         self._write_item = self.pipe_writer.write_item
-        kwargs = {}
         if SCRAPY_VERSION_INFO < (2, 11):
-            kwargs["binary"] = False
-        self.exporter = PythonItemExporter(**kwargs)
+            with ignore_warnings(category=ScrapyDeprecationWarning):
```

Comment:
The warning is only for binary=True, the old code had ignore_warnings() because previously it was `exporter_kwargs = {'binary': False} if not IS_PYTHON2 else {}`

Reply:
Oh, nice!